### PR TITLE
Add timestamps for video 3: Violeta keynote

### LINKED ISSUE #13: working on the timestamps of videos-list/03-viola.md
To avoid that several people are working on the same video I opened that issue.  It can be closed when the timestamps are committed.

[videos-list/03-viola.md](https://github.com/pymc-devs/video-timestamps/blob/main/videos-list/03-viola.md)

### PR DESCRIPTION
**Pull request by bsenst**
### Reference
Towards #11 
Closes #13

### Description
Add timestamps for video 3: Violeta keynote

### CODE REVIEW

**Comment by twiecki** [comment]
Thanks for the contribution @bsenst!

**Review by BerylKanali — APPROVED** [review]
@bsenst Thank you for your contribution, the timestamps look good.
cc: @twiecki

**Comment by reshamas** [comment]
@bsenst Thanks for the PR.  Please see the below review and feedback:
- Notice that the PR description at very top which I edited is more informative than the previous "add timestamps"
- In description, I added reference to the issues:
```
### Reference
Towards #11 
Closes #13

### Description
Add timestamps for video 3: Violeta keynote
```

- There is no need for you to manually close issue3.  Once this PR is reviewed, approved and merged, it will automatically be closed, as long as it has been linked at the top, one of the perks of this powerful GitHub workflow.

File: videos-list/03-viola.md [hunk 1]
```diff
@@ -13,12 +13,41 @@ Discourse Discussion
...
 ## Timestamps
-- 0:00 Start of event
-- x:xx 
-- x:xx
-
-## Note: help us add timestamps here
-https://github.com/pymc-devs/video-timestamps
+00:00:00 Introductory words by Viola
+00:00:48 The daily new confirmed COVID-19 cases
+00:01:39 The agenda of the talk
```
Comment (reshamas) [inline videos-list/03-viola.md:18]:
```suggestion
00:00 Introductory words by Viola
00:48 The daily new confirmed COVID-19 cases
01:39 The agenda of the talk
```

Comment (reshamas) [reply videos-list/03-viola.md:18]: @bsenst 
For videos that are less than 1 hour long, we do not need the first two "00:".  Can we remove here and also for the rest of the timestamps in this file?  Thank you.

**Comment by reshamas** [comment]
Confirming that I have:  
-[x] added the timestamps to the video description here: https://youtu.be/btHkSlePV5Q
-[x] updated the checklist on issue #11 to indicate this video is complete

Thank you @bsenst for your contribution to the Data Umbrella community and PyMC project. 

#DataUmbrellaPyMCSprint

**Pull request merged**